root/edit/details/historic/AddTrackKV.js

```diff
@@ -35,6 +35,13 @@ type Props = {
 const AddTrackKV = ({edit}: Props): React.Element<'table'> => {
   const display = edit.display_data;
   const artist = display.artist;
+  /*
+   * Some lengths of -1 or 0 ms are stored, which is nonsensical
+   * and probably meant as a placeholder for unknown duration
+   */
+  const length = (display.length && display.length <= 0)
+    ? null
+    : display.length;
```

Comment:
If `display.length` is 0, won't `display.length && display.length <= 0` be false unless it's a string?

Reply:
I guess that's a fair point, should probably just do `display.length != null` then?

Reply:
In place of the first check yeah, that should work.